ccd-definition/AuthorisationCaseField.json

```diff
@@ -1083,5 +1090,12 @@
     "CaseFieldID": "dateSubmitted",
     "UserRole": "caseworker-publiclaw-gatekeeper",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "applicant",
+    "UserRole": "caseworker-publiclaw-gatekeeper",
+    "CRUD": "CRU"
```

Comment:
permissions for gatekeeper applicant need to be reverted to R

Reply:
👍